drivers/dose/Makefile.dep

```diff
@@ -2,6 +2,10 @@ FEATURES_REQUIRED += periph_gpio_irq
 FEATURES_REQUIRED += periph_uart
 FEATURES_OPTIONAL += periph_uart_rxstart_irq
 
+ifneq (,$(filter dose_watchdog,$(USEMODULE)))
+  FEATURES_REQUIRED += periph_timer_periodic
+endif
```

Comment:
Can this get rid of the xtimer dependency?

Reply:
No xtimer is still used for the random backoff and as fallback if no HW timer is configured